CLI/add_feature_extractor.py

```diff
@@ -79,6 +79,11 @@ def parser_function() -> argparse.ArgumentParser:
         sys.exit(-1)
     extractor_target_path.mkdir()
     shutil.copytree(extractor_source, extractor_target_path, dirs_exist_ok=True)
+
+    # Set execution permissions for wrapper
+    sparkle_extractor_wrapper = extractor_target_path / gv.sparkle_extractor_wrapper
+    if sparkle_extractor_wrapper.exists():
+        sparkle_extractor_wrapper.chmod(0o755)
```

Comment:
Don't change the executable permissions here, simply check if we have them and if not, print error and sys exit -1. For any sparkle extractor wrappers that we deliver for the user, we make sure that this is already done in the terminal with chmod +x